### ISSUE TITLE
RadarChartView the datas disappear when I enlarge or decrease the window

### ISSUE THREAD

**Issue by thierryH91200** [danielgindi/Charts#2207]
In my program inspired by ios demo
This is what happens

https://www.dropbox.com/s/r4bmn262esatiuf/screencast%202017-02-28%2010-31-41.mp4?dl=0

If in my program I put in comment the next line?

>      yAxis.axisMaximum = 80.0

 

>       let yAxis = radarChartView.yAxis
>         yAxis.labelFont = NSUIFont(name: "HelveticaNeue-Light", size: CGFloat(9.0))!
>         yAxis.labelCount = 5
>         yAxis.axisMinimum = 0.0
>        //yAxis.axisMaximum = 80.0
>         yAxis.drawLabelsEnabled = false
> 

https://www.dropbox.com/s/n1vvg4e37m5psxv/screencast%202017-02-28%2010-33-20.mp4?dl=0

OK the problem is solved but why ??

**Comment by liuxuan30** [comment]
eh... seems a bug

**Comment by liuxuan30** [comment]
I looked into it:
When setting `yAxis.axisMaximum`, 
in `open override func notifyDataSetChanged()`,  the input is 
```swift
        _yAxisRenderer?.computeAxis(min: _yAxis._axisMinimum, max: _yAxis._axisMaximum, inverted: _yAxis.isInverted)
```
however inside `computeAxis()` -> `computeAxisValues()` in `YAxisRendererRadarChart`,  `computeAxisValues()` is changing axis.entries and updating it after getting new entires:
```swift
        axis._axisMinimum = axis.entries[0]
        axis._axisMaximum = axis.entries[n-1]
        axis.axisRange = abs(axis._axisMaximum - axis._axisMinimum)
```
And here come's the dead loop, axisMaximum is increasing forever, causing the radar chart disappeared.

I remember I modified the code here a long time ago in https://github.com/danielgindi/Charts/pull/207,
however, I just add `_yAxis.axisMinimum = _yAxis.entries[0]`, axis._axisMaximum = axis.entries[n-1] is there all the time. 
Looks like an old bug, but I'm not sure if Chart 2.x has the same issue or it happen for Chart 3.x

**Comment by thierryH91200** [comment]
Thanks for the explanation
I understand better now

**Comment by liuxuan30** [comment]
yep, it just keeps increasing no matter you how you change the window.
However I need @danielgindi to give some thoughts why it's implemented this way at that moment, and for chart 3.x, is it possible to remove it.
